src/renderer/component/speechUpload/view.jsx

```diff
@@ -0,0 +1,86 @@
+import React from "react";
```

Comment:
I think this and other files should be called something along the lines of `thumbnailSelect` and not `speechUpload`